packages/lit-html/src/lib/lit-html.ts

```diff
@@ -213,33 +222,35 @@ class Template {
     const commentRegex = /-->/g;
     // Comments not started with <!--, like </{, can be ended by a single `>`
     const comment2Regex = />/g;
-    const tagRegex = /(>)|(?:(?:[ \x09\x0a\x0c\x0d])([^\0-\x1F\x7F-\x9F "'>=/]+)([ \x09\x0a\x0c\x0d]*=[ \x09\x0a\x0c\x0d]*([^ \x09\x0a\x0c\x0d"'`<>=]|("|')|$)))/g;
+    const tagRegex = new RegExp(TAG_STATE_REGEX, 'g');
```

Comment:
Can we initialize this once at the module root?

Reply:
Done